lang/java/reef-common/src/main/java/org/apache/reef/util/MemoryUtils.java

```diff
@@ -96,6 +96,7 @@ public static void resetPeakUsage() {
     }
   }
 
+  @SuppressWarnings("checkstyle:avoidnestedblocks")
```

Comment:
It looks like the nested block can safely be removed, and then we won't want this suppressed.